src/js/components/AnimatedLogo.js

```diff
@@ -24,7 +23,7 @@ var AnimatedLogo = React.createClass({
 
   componentWillMount: function () {
     this.internalStorage_set({
-      logoGradientID: _.uniqueId('logoGradient'),
+      logoGradientID: `logoGradient-${Math.random()}`,
```

Comment:
This doesn't quite guarantee a unique ID. It would be nice if we created a util for this and not have it using floats.